application/factory.py

```diff
@@ -239,7 +239,11 @@ async def add_x_content_type_options_header(request: Request, call_next):
     app.add_middleware(SuppressClientDisconnectNoResponseReturnedMiddleware)
 
     if settings.SENTRY_DSN:
-        sentry_sdk.init(dsn=settings.SENTRY_DSN, environment=settings.ENVIRONMENT)
+        sentry_sdk.init(
+            dsn=settings.SENTRY_DSN,
+            environment=settings.ENVIRONMENT,
+            traces_sample_rate=0.1,
```

Comment:
I wonder if it's worth making this a `setttings` attribute populated via environment variable?

Reply:
Yeah thats the follow up in the description. This is just to make sure it functions as expected 👍